### PR TITLE
Add Game Developments Courses / Badges🏅

### PR DESCRIPTION
**Pull request by PrathamArtz**

### CODE REVIEW

**Comment by PrathamArtz** [comment]
unicorn

File: .vs/Open_source/v17/.wsuo
Comment (waleedsadek-panx) [inline .vs/Open_source/v17/.wsuo:1]: Delete this file

File: .vs/Open_source/v17/DocumentLayout.json
Comment (waleedsadek-panx) [inline .vs/Open_source/v17/DocumentLayout.json:1]: Delete this file

File: .vs/slnx.sqlite
Comment (waleedsadek-panx) [inline .vs/slnx.sqlite:1]: Delete this file

File: README.md
Comment (waleedsadek-panx) [inline README.md:1]: - [ ] On line 62 change `- [Game Devlopement](#Game-developement)` to `- [Game Development](#game-development)`  
- [ ] Delete line 90 and 365
- [ ] On line 374 change `## Game-developement` to `## Game Development`
- [ ] Delete line 375 and 376

**Review by waleedsadek-panx — CHANGES_REQUESTED** [review]
Thanks for your contribution! 🚀 Kindly make these amends, so I can merge your request! Let me know if you need anything.

- [ ]  On line 62 change `- [Game Devlopement](#Game-developement)` to `- [Game Development](#game-development)` and add it before `[Other Awesome Lists](#other-awesome-lists)`
- [ ] Delete line 90 and 365
- [ ] On line 374 change `## Game-developement` to `## Game Development`
- [ ] Delete line 375 and 376
- [ ] Delete file `.vs/Open_source/v17/.wsuo`
- [ ] Delete file `.vs/Open_source/v17/DocumentLayout.json`
- [ ] Delete file `.vs/VSWorkspaceState.json`
- [ ] Delete file `.vs/slnx.sqlite`

**Comment by waleedsadek-panx** [comment]
Thanks for hunting down these new certificates! I left a couple of notes before I can merge.

File: README.md [hunk 1]
```diff
@@ -325,7 +327,6 @@ Report it [here](https://github.com/PanXProject/awesome-certificates/issues/new?
 | [Intro to Programming](https://www.kaggle.com/learn/intro-to-programming)           | Kaggle          | Beginner      | 5     | 🏆     |
 | [Python for Machine Leaning and Data Science](https://www.mygreatlearning.com/academy/learn-for-free/courses/python-for-machine-learning-and-data-science) | Great Leaning |Beginner| 3 | 🏆 |
 
-
```
Comment (waleedsadek-panx) [inline README.md:328]: Add space so it's two lines after every table.

**Pull request closed without merging**